odmantic/engine.py

```diff
@@ -369,11 +385,15 @@ async def save_all(self, instances: Sequence[ModelType]) -> List[ModelType]:
             The save_all operation actually modify the arguments in place. However, the
             instances are still returned for convenience.
         """
-        async with await self.client.start_session() as s:
-            async with s.start_transaction():
-                added_instances = await asyncio.gather(
-                    *[self._save(instance, s) for instance in instances]
-                )
```

Comment:
Seems odd that this used to work on all types of servers. Since creating a transaction on standalone should not have worked.